### PR TITLE
Commits should show the date as well

### LINKED ISSUE #68: Commits should show the date as well
This is helpful in quickly knowing if any changes have occurred to the branch.

image

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #68

### CODE REVIEW

**Comment by alexr00** [comment]
image

File: webviews/components/timeline.tsx [hunk 1]
```diff
@@ -55,6 +55,8 @@ const CommitEventView = (event: CommitEvent) =>
 			<a className='message' href={event.htmlUrl}>{event.message}</a>
 		</div>
 		<a className='sha' href={event.htmlUrl}>{event.sha.slice(0, 7)}</a>
+		{nbsp}
+		<div>{event.authoredDate}</div>
```
Comment (RMacfarlane) [inline webviews/components/timeline.tsx:59]: There's a `Timestamp` component we could use here to show the date formatted as "X days ago" which I think is more consistent with how other dates are displayed

**Review by RMacfarlane — APPROVED** [review]
Looks good!

File: webviews/components/timeline.tsx (hunk 1 above)
Comment (alexr00) [reply webviews/components/timeline.tsx:59]: Works great, thanks for the pointer!

**Pull request merged**